include/zephyr/drivers/pcie/pcie.h

```diff
@@ -36,6 +37,95 @@ typedef uint32_t pcie_bdf_t;
  */
 typedef uint32_t pcie_id_t;
 
+struct pcie_dev {
+	pcie_bdf_t bdf;
+	pcie_id_t  id;
+};
+
+#define Z_DEVICE_PCIE_NAME(node_id) _CONCAT(pcie_dev_, DT_DEP_ORD(node_id))
+
+/**
+ * @brief Get the PCIe Vendor and Device ID for a node
+ *
+ * @param node_id DTS node_id
```

Comment:
I would change it to `DTS node identifier`

Reply:
Fixed, thanks